charts/sourcegraph/examples/kustomize-chart/README.md

```diff
@@ -0,0 +1,17 @@
+# kustomize chart
+
+This example demonstrate the escape hatch whenever you encounter something that our helm charts do not support yet. [Kustomize] allows you to apply [strategic merge patch] and [json patch] on helm chart without maintaining a custom fork. This is our **recommended** approach to customize helm chart, and you should avoid forking unless absolutely necessary.
+
+## Customize `frontend` service port
+
+In this example, we will change the `http` port of `sourcegraph-frontend.Service.yaml` and update the corresponding backend service port number in `sourcegraph-frontend.Ingress.yaml`.
+
+> Below command should be run within the current directory
+
+```sh
+helm upgrade --install --create-namespace -n sourcegraph sourcegraph sourcegraph/sourcegraph --post-renderer ./kustomize
```

Comment:
Might be worth adding some extra explanation about the `./kustomize` script and its role in this process